HW1_DiceGame/app/src/main/java/edu/fullerton/csu/team06/hw1_dicegame/Gameplay_UI.java

```diff
@@ -0,0 +1,83 @@
+package edu.fullerton.csu.team06.hw1_dicegame;
+
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.Spinner;
+import android.widget.Toast;
+
+import java.util.Random;
+
+public class Gameplay_UI extends AppCompatActivity {
+    Spinner spinner;
+    ArrayAdapter<CharSequence> adapter;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_gameplay__ui);
+
+        // Get the Intent that started this activity
+        Intent intent = getIntent();
+
+        spinner = (Spinner)findViewById(R.id.spinner);
+        adapter = ArrayAdapter.createFromResource(this,R.array.dice_values,android.R.layout.simple_spinner_item);
+        adapter.setDropDownViewResource(android.R.layout.simple_spinner_item);
+        spinner.setAdapter(adapter);
+
+        spinner.setOnItemSelectedListener(new AdapterView.OnItemSelectedListener() {
+            @Override
+            public void onItemSelected(AdapterView<?> parent, View view, int position, long id) {
+                if  (position > 1) {
+                    Toast.makeText(getBaseContext(), parent.getItemIdAtPosition(position) + " selected", Toast.LENGTH_SHORT).show();
+                    Player_choose(parent);
+                }
+            }
+
+            @Override
+            public void onNothingSelected(AdapterView<?> parent) {
+
+            }
+        });
+    }
+
+    /** Called when the user taps either of the CPU dice */
+    public void CPU_roll(View view) {
```

Comment:
2 things regarding the roll functions:

1. I think we should have a generic `roll(View view)` function for both the Player and the Computer which sets the value in the view that is passed in (and eventually the player who will also be passed in)
2. Watch casing. Capital first letters represent Classes while camelCase represent functions. Underscores are usually a C++/C thing or for constants. See https://www.ibm.com/developerworks/library/j-perry-writing-good-java-code/index.html

Reply:
I agree with one function, waiting until team meeting to make any changes to this.

Reply:
to minimize reuse, couldn't the simulate dice roll be a function call within the Player class? something like:
value = plyr.simRoll
For cpu it returns both values or just do it twice and player would always do this for dice 2.
No matter what the smarts of who is rolling and how its assigned needs to be somewhere and I would think at the ui level the smarts should only be about who.  This, of course, adds more work to the Player class, but I think it makes sense.

Reply:
I agree with that. I think it makes a lot of sense to have the player doing the rolling. As apposed to the game do the rolling and assigning the roll value to the player.

Reply:
I'm fine with that. That way we can keep the logic out of the Gamplay Activity and have the Player object deal with the rolling.